apps/workflows-dashboard/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:18.16-alpine3.17 as dev
+
+WORKDIR /app
+
+COPY . .
+
+RUN mv /app/.env.example /app/.env; rm -rf /app/node_modules; npm install --legacy-peer-deps ; npm run build
+
+ENV PATH="$PATH:./node_modules/.bin"
+
+EXPOSE 5200
+
+CMD ["npm", "run", "dev", "--host"]
```

Comment:
this should run build (creating production artifcagts)

Reply:
Yes, we are building [here](https://github.com/ballerine-io/ballerine/pull/604/files/002fd2806a81f6f4ede1575f2ff69f20af951786#diff-36bec7eba451b2f6f6608eca0e87e2ce93131549e494bde47daa020da9f8329cR7) 

this is a multi-stage build and if a engineer wants a dev image we now give that. 
eg:
if a engineer does 
`docker build . ` --> this will create prod image 
`docker build . --target dev `. ---> this will create a dev image with all code, dist, node_modules present and helps engineer in faster debugging.